flow-api/src/main/java/com/sequenceiq/flow/api/FlowEndpoint.java

```diff
@@ -80,4 +86,23 @@ public interface FlowEndpoint {
     @ApiOperation(value = "Check if there is a running flow for flow id", produces = "application/json", notes = "Flow log operations",
             nickname = "hasFlowRunningByFlowId")
     FlowCheckResponse hasFlowRunningByFlowId(@PathParam("flowId") String flowId);
+
+    @GET
+    @Path("/check/flowIds")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = "Get flow logs by a list of flow ids - Input size max 50", produces = "application/json", notes = "Flow log operations",
+            nickname = "getFlowLogStatusesByFlowIds")
+    List<FlowLogResponse> getFlowLogStatusesByFlowIds(@NotNull @Size(max = 50, message = "Input size should not be over 50")
+        @QueryParam("flowIds") List<String> flowIds,
+        @Min(value = 1, message = "Page size should be greater than 0") @QueryParam("size") int size, @QueryParam("page") int page);
+
+    @GET
+    @Path("/check/chainIds")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = "Check if there is a running flow for chain ids - Input size max 50",
+            produces = "application/json", notes = "Flow check log operations",
+            nickname = "getFlowChainsStatusesByChainIds")
+    List<FlowCheckResponse> getFlowChainsStatusesByChainIds(@NotNull @Size(max = 50, message = "Input size should not be over 50")
+        @QueryParam("chainIds") List<String> chainIds,
+        @Min(value = 1, message = "Page size should be greater than 0") @QueryParam("size") int size, @QueryParam("page") int page);
```

Comment:
We are not sending any paging information in response. how would client know if it needs to make subsequent calls or not? We should send link for next page in response so that client knows if there are more records to fetch or not.

Reply:
addressed.pls verify